controller/src/mantarray_desktop_app/main_process/server.py

```diff
@@ -1104,11 +1100,13 @@ def after_request(response: Response) -> Response:
                 response_json["root_recording_path"]
             )
     msg = "Response to HTTP Request in next log entry: "
-    if response.status_code == 200:
+    if response.status_code == 200 and "system_status" not in rule.rule:
         # Tanner (1/19/21): using json.dumps instead of an f-string here allows us to perform better testing of our log messages by loading the json string to a python dict
         msg += json.dumps(response_json)
-    else:
+    elif response.status_code != 200:
         msg += response.status
+    else:
+        return response
     logger.info(msg)
```

Comment:
This will still log /system_status. What I'm getting at is that you only really need to do this here
```
if not ("system_status" in rule.rule and response.status_code == 200):
    logger.info(msg)
```
The `and "system_status" not in rule.rule:` is unnecessary